<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCaptureCannyClass</class>
 <widget class="QWidget" name="VideoCaptureCannyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoCaptureCanny</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="loadVideo_button">
     <property name="text">
      <string>Load Video</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="camera_button">
     <property name="text">
      <string>Camera Capture</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="canny_button">
       <property name="text">
        <string>Canny</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cannyA_text"/>
     </item>
     <item>
      <widget class="QLineEdit" name="cannyB_text"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
